modules/issue_tracker/test/issue_tracker_test_plan.md

```diff
@@ -1,17 +1,17 @@
 # Issue Tracker Test Plan
 
 ## Issue Tracker Filter Form [Automation Testing]
-1. User can access the page only if they have `issue_tracker_reporter` or `issue_tracker_developer` permission
+1. User can access the page only if they have `Issue Tracker: Close/Edit/Re-assign/Comment on All Issues` or ` Issue Tracker: Create/Edit Own Issues and Comment on All Issues` permission
```

Comment:
@charliehenrib these will need to still include the backend names. you can add them in parenthesis if you want.

Reply:
choose a convention like the one below and stick to it maybe

```
_Front end label name_ (`backend_permission`)
```